java/README.md

```diff
@@ -1,17 +1,36 @@
-# Java Bindings Development
+# Vortex Java bindings
 
-Install a JDK 17 toolchain.
+We provide two interfaces for working with Vortex from Java:
 
-You can run tests like
+- `vortex-java` - a low-level interface JNI for working with Vortex files and arrays on cloud and local storage
+- `vortex-spark` - A Spark connector for working with datasets of Vortex files
 
-```
-./gradlew test
-```
+## Publishing
+
+We publish three artifacts out of this repo at CI time to Maven Central Sonatype:
 
-You can build a JAR file with
+* `vortex-jni` JAR containing the JNI code, plus compiled native libraries for all of the following targets: `aarch64-apple-darwin`, `aarch64-unknown-linux-gnu`, `x86_64-unknown-linux-gnu`
+* `vortex-jni-all` which is the "shadow JAR" containing all of `vortex-jni` as well as all upstream Java dependencies packaged in a single JAR.
+* `vortex-spark` which is the runtime JAR needed for the Vortex Spark bindings
+
+We use the [following GPG key](https://keyserver.ubuntu.com/pks/lookup?search=8745D1A87C0B2159&fingerprint=on&op=index) for publishing:
 
 ```
-./gradlew jar
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Interestingly, Maven Central does nothing to verify that you own the key that you sign with, they just make sure that your key is visible from one of the "well-known" OpenPGP servers. I uploaded to ubuntu and MIT keyservers to be safe